libs/payments/customer/src/lib/subscription.manager.ts

```diff
@@ -38,7 +38,8 @@ export class SubscriptionManager {
 
   async update(
     subscriptionId: string,
-    params?: Stripe.SubscriptionUpdateParams
+    params?: Stripe.SubscriptionUpdateParams,
+    metadata?: Stripe.Metadata
```

Comment:
`metadata` is already a property of the `params: Stripe.SubscriptionUpdateParams` argument, why include a separate argument specifically for `metadata`?

Reply:
This should all go away now with the update to `STRIPE_SUBSCRIPTION_METADATA`.